### PR TITLE
Drop Appveyor in favor of Github actions.

### LINKED ISSUE #748: Appveyor CI sometimes fails
As #746 check failed unexpectedly.

### PR DESCRIPTION
**Pull request by dzhuang**
Fix #748 . Closing #747 #748 .

### CODE REVIEW

**Comment by dzhuang** [comment]
@inducer You'll need to remove this project from [your Appveyor projects](https://ci.appveyor.com/projects).

**Review by joshuata — APPROVED** [review]
Looks great. I'll see if I can figure out why that test is throwing exceptions in GH actions and not appveyor

**Comment by dzhuang** [comment]
@joshuata Thank you for your previous work on this!

Thanks!
>Looks great. I'll see if I can figure out why that test is throwing exceptions in GH actions and not appveyor

File: appveyor.yml [hunk 1]
```diff
@@ -1,74 +0,0 @@
...
-cache:
-  - '%LOCALAPPDATA%\pip\Cache'
-  - memcached -> appveyor.yml
-
-environment:
-  global:
-    # SDK v7.0 MSVC Express 2008's SetEnv.cmd script will fail if the
-    # /E:ON and /V:ON options are not enabled in the batch script intepreter
-    # See: http://stackoverflow.com/a/13751649/163740
-    CMD_IN_ENV: "cmd /E:ON /V:ON /C .\\.ci\\run_with_env.cmd"
```
Comment (inducer) [inline appveyor.yml:19]: `run_with_env.cmd` doesn't seem to be used any more. Delete?

**Review by inducer — COMMENTED** [review]
Thanks for your work on this. Some comments below.

File: tests/test_versioning.py [hunk 2]
```diff
@@ -249,8 +249,11 @@ def test_set_up_new_course_git_source_invalid(self):
             self.assertAddMessageCalledWith(
                 "No refs found in remote repository")
 
+    @pytest.mark.slow
```
[the remark this answers is not in the document]
Comment (dzhuang) [reply tests/test_versioning.py:252]: Good to know that. Thanks.

File: appveyor.yml (hunk 1 above)
Comment (dzhuang) [reply appveyor.yml:19]: Sure.

Comment (dzhuang) [reply appveyor.yml:19]: P.S., I did try to look for this file.

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -119,39 +119,32 @@ jobs:
...
-  #       with:
-  #         fail_ci_if_error: true
+  pytest-windows:
+    name: Python - Windows
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v1
+        with:
+          python-version: 3.7
```
[the remark this answers is not in the document]
Comment (dzhuang) [reply .github/workflows/ci.yml:129]: I'm ok for 3.8.

**Comment by inducer** [comment]
Thanks for working on this!

**Comment by dzhuang** [comment]
@inducer FYI, you need to remove this project from [your Appveyor projects](https://ci.appveyor.com/projects) for tests to pass.

**Comment by inducer** [comment]
> @inducer FYI, you need to remove this project from [your Appveyor projects](https://ci.appveyor.com/projects) for tests to pass.

Thanks, done.

**Pull request merged**